bindata/etcd/etcd-member-remove.sh

```diff
@@ -17,21 +17,40 @@ if [ "$1" == "" ]; then
     usage
 fi
 
-ETCD_NAME=$1
-ASSET_DIR=/home/core/assets
-ASSET_DIR_TMP="$ASSET_DIR/tmp"
-ETCDCTL=$ASSET_DIR/bin/etcdctl
-ETCD_DATA_DIR=/var/lib/etcd
-CONFIG_FILE_DIR=/etc/kubernetes
+source etcd.env
+
+# download etcdctl from upstream release assets
+function dl_etcdctl() {
+  ETCDCTL_BIN="${ASSET_DIR}/bin/etcdctl"
+  local etcdimg="registry.svc.ci.openshift.org/ci-op-l7y8kffx/stable@sha256:c576980bdce812e5d000e6f96fc238d75688da01f767cdaed6cbf04c40c600fa"
+  local etcdctr=$(podman create ${etcdimg})
+  local etcdmnt=$(podman mount "${etcdctr}")
+  cp ${etcdmnt}/bin/etcdctl $ASSET_DIR/bin
+  umount "${etcdmnt}"
+  podman rm "${etcdctr}"
+  $ETCDCTL_BIN version
+}
+
+function etcd_member_remove() {
+  NAME="$1"
 
-source "/usr/local/bin/openshift-recovery-tools"
+  ID=$($ETCDCTL_WITH_TLS member list | awk -F "," "/\s${NAME}\,/"'{print $1}') || true
+  if [ "$?" -ne 0 ] || [ -z "$ID" ]; then
+    echo "could not find etcd member $NAME to remove."
+    exit 1
```

Comment:
we normally run our scripts with something that makes them fail if the command fails.  I'd prefer the script have that

Reply:
Yes. I will add `-e`.